daemon/control.c

```diff
@@ -53,6 +53,12 @@
 
 #include <google/protobuf-c/protobuf-c-text.h>
 
+//Needed for log copy functionality
+#include <dirent.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+
```

Comment:
in general use libcommon for file/dir operations

Reply:
libcommon is now used for file/dir operations.